pom.xml

```diff
@@ -1048,8 +1040,11 @@
                 </plugins>
             </build>
         </profile>
+        <!-- Adds all zip build artifacts to the OWASP scan. This includes the kernel, the DDF
+             distro, etc. Those may include artifacts which aren't actually dependencies of any
+             DDF modules, so won't appear in a normal scan. -->
         <profile>
-            <id>owasp</id>
+            <id>owasp-dist</id>
```

Comment:
❓Why not just always scan the zips? Is it because it takes longer to run?

Reply:
I just did it this way because that's how it was before. I'm open to changing it if we want